packages/vulcan-lib/lib/modules/errors.js

```diff
@@ -84,13 +84,15 @@ export const getErrors = error => {
   if (error.graphQLErrors && error.graphQLErrors.length > 0) {
     // get graphQL error (see https://github.com/thebigredgeek/apollo-errors/issues/12)
     const graphQLError = error.graphQLErrors[0];
-    if (graphQLError.data && !isEmpty(graphQLError.data)) {
-      if (graphQLError.data.errors) {
+    const data = (graphQLError && graphQLError.extensions && graphQLError.extensions.exception && graphQLError.extensions.exception.data) ||
```

Comment:
```suggestion

    const data = (graphQLError && graphQLError.extensions && graphQLError.extensions.exception && graphQLError.extensions.exception.data) ||
```

Reply:
Nope, doesn't work, because we are in the Vulcan package which doesn't currently have `?.` support